zummon/src/commonMain/kotlin/companysurvey/TimeSeries.kt

```diff
@@ -27,10 +27,10 @@ data class TimeSeries (
     val start: Instant,
     val timeStep: kotlin.time.Duration = 15.minutes,
     val unit: TimeSeriesUnit = TimeSeriesUnit.KWH,
-    val values: FloatArray = floatArrayOf(),
+    val values: List<Float?> = listOf(),
```

Comment:
`List<Float>` does a memory allocation for each of the ~35040 values. It is a lot slower.

Also this is a breaking change in the AnyLogic client which I prefer to avoid.

If this is about wanting to store null, we don't need it because there is `Double.NaN`.

Reply:
How do you get to know the performance?